### PR TITLE
Feature/project 1

### PR DESCRIPTION
**Pull request by RoseKoning**

### CODE REVIEW

File: staff/rose-koning/project-1/website-1/about.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body class="grid">
+    <header class="header">
+        <h1>
+            <img src="assets/nysl_logo.png" alt="North Side Youth Soccer League Logo" class="logo" />North Side Youth
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-1/about.html:14]: should an img be nested inside an h1?

Comment (RoseKoning) [reply staff/rose-koning/project-1/website-1/about.html:14]: I googled it. It seems to be a quick-fix. I guess if it´s semantically correct it shouldn´t be and I could use flex box inside of the header to have the title element and logo next to each other?

File: staff/rose-koning/project-1/website-4/home.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            North Side Youth Soccer League<img src="assets/nysl_logo.png" alt="nysl logo" class="logo" />
+        </h1>
+    </header>
+    <section class="picbar"></section>
+    <section class="pagebar">
+        <h2 class="pagebar h2">Home</h2>
+    </section>
+    <nav class="navbar">
+      <ul class="ul">
+        <a href="home.html" target="_self">
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html]: should ul wrap li elements only, and should li elements wrap elements for each item, in this case anchors, but not the opposite. so should anchors be moved inside and then be contained by list items

File: staff/rose-koning/project-1/website-4/home.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        </a>
+        <a href="registration.html" target="_self">
+            <li class="navbar navbutton">Registration</li>
+        </a>
+        <a href="contact.html" target="_self">
+            <li class="navbar navbutton">Contact</li>
+        </a>
+    </ul>
+      </nav>
+    <div class="info">
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html]: this selector doesn't seem to have a block, may you wanted to say `grid__info-subtitle`instead? (grid as the block for this element)

File: staff/rose-koning/project-1/website-4/home.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      <img src="assets/soccer_image1.jpg" class="image">
+        <h2>Upcoming events</h2>
+        <h3>August 4th</h3>
+        <p>NYSL fundraiser</p>
+        <h3>August 16th</h3>
+        <p>Season kick-off: meet the teams</p>
+        <h3>September 1st</h3>
+        <p>First game of the season(check game schedule for specifics)</p>
+      </div>
+    <div class="image">
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html]: should this image be part of the content as an `img` tag instead of a `div` with a background?

**Review by manuelbarzi — CHANGES_REQUESTED** [review]

File: staff/rose-koning/project-1/website-4/home.html (hunk 2 above)
Comment (RoseKoning) [reply staff/rose-koning/project-1/website-4/home.html]: done

File: staff/rose-koning/project-1/website-4/home.html (hunk 4 above)
Comment (RoseKoning) [reply staff/rose-koning/project-1/website-4/home.html]: this part no longer exists

File: staff/rose-koning/project-1/website-4/home.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+<body class="grid">
+    <header class="header">
+        <h1 class="header h1">
+            North Side Youth Soccer League<img src="assets/nysl_logo.png" alt="nysl logo" class="logo" />
+        </h1>
+    </header>
+    <section class="picbar"></section>
+    <section class="pagebar">
+        <h2 class="pagebar h2">Home</h2>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html]: this selector doesn't seem to have a block, may you wanted to say `grid__pagebar-title`instead? (grid as the block for this element)

File: staff/rose-koning/project-1/website-4/home.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body class="grid">
+    <header class="header">
+        <h1 class="header h1">
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html]: this selector doesn't seem to have a block, may you wanted to say `grid__header-title`instead? (grid as the block for this element)

File: staff/rose-koning/project-1/website-4/home.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            <li class="navbar navbutton">NYSL Rules</li>
+        </a>
+        <a href="registration.html" target="_self">
+            <li class="navbar navbutton">Registration</li>
+        </a>
+        <a href="contact.html" target="_self">
+            <li class="navbar navbutton">Contact</li>
+        </a>
+    </ul>
+      </nav>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-1/website-4/home.html:45]: this selector doesn't seem to have a block, may you wanted to say `grid__info-title`instead? (grid as the block for this element)

File: staff/rose-koning/project-1/website-4/home.html (hunk 3 above)
Comment (RoseKoning) [reply staff/rose-koning/project-1/website-4/home.html]: Yes! it was changed while I was updating to bem

**Review by manuelbarzi — CHANGES_REQUESTED** [review]

File: staff/rose-koning/project-1/website-4/home.html (hunk 6 above)
Comment (RoseKoning) [reply staff/rose-koning/project-1/website-4/home.html]: updated in bem

**Pull request merged**